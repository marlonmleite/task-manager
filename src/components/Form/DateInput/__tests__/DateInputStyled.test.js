import React from 'react'
import { shallow } from 'enzyme'
import Date from '../styled'

describe('DateInputStyled', () => {
  it('should render', () => {
    const date = shallow(<Date />)

    expect(date).toMatchSnapshot()
  })
})
