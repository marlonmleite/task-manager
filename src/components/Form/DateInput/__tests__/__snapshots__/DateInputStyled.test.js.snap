// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateInputStyled should render 1`] = `
.c0 .react-datepicker {
  border-radius: 2px;
  font-family: inherit;
}

<div
  className="c0"
/>
`;
